tracer/test/test-applications/integrations/Samples.GraphQL7/Samples.GraphQL7.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
```

Comment:
Maybe it's because you've iterated on this a bunch, but it seems like if you write the correct conditions you could merge this into Samples.GraphQL4.csproj so there's one less csproj to deal with. If it's not too much hassle, I think it would be worth the effort

Reply:
If you do this, you would have to change the integration test (GraphQLTests) to make sure that if v7 is used then you use the new `GraphQL7Tests` snapshot

Reply:
That was my initial preference/approach too, [but as pointed out here](https://github.com/DataDog/dd-trace-dotnet/pull/3842#discussion_r1129308598), that doesn't work because we want to test against _both_ graphql 4 and 7 on Windows too, where we don't do the multi-package testsing (yet). Without two samples, we can't do that (or it requires a lot of work!) so this seemed like the best middle ground